Classes/Configuration/ExtensionBuilderConfigurationManager.php

```diff
@@ -117,7 +117,7 @@ public function getSettings($typoscript = null)
         $settings = $typoscript['module.']['extension_builder.']['settings.'];
         $settings['extConf'] = $this->getExtensionBuilderSettings();
         if (empty($settings['publicResourcesPath'])) {
-            $settings['publicResourcesPath'] = ExtensionManagementUtility::siteRelPath('extension_builder') . 'Resources/Public/';
+            $settings['publicResourcesPath'] = ExtensionManagementUtility::extPath('extension_builder'). 'Resources/Public/';
```

Comment:
This is an absolute path now, but was a relative path before. I'm not sure whether this change is safe.